templates/rancher/overlay/opt/corral/rancher/install-cert-manager.sh

```diff
@@ -9,6 +9,8 @@ helm upgrade \
   --create-namespace \
   -n cert-manager \
   --set installCRDs=true \
-  --version v1.5.0 \
+  --version v1.8.0 \
   --wait \
-  cert-manager jetstack/cert-manager
\ No newline at end of file
+  cert-manager jetstack/cert-manager
+
+sleep 1m
```

Comment:
Why do we need to sleep here?

Reply:
I have issues when I attempt to install rancher right after installing cert-manager and the installation fails. But if I give it enough time, it will install with no issue.